app/scripts/metamask-controller.js

```diff
@@ -217,6 +217,7 @@ import { STATIC_MAINNET_TOKEN_LIST } from '../../shared/constants/tokens';
 import { getTokenValueParam } from '../../shared/lib/metamask-controller-utils';
 import { isManifestV3 } from '../../shared/modules/mv3.utils';
 import { convertNetworkId } from '../../shared/modules/network.utils';
+
```

Comment:
This file doesn't need changes... remove whitespace changes.

Lets connect this controller in the UI integration PRs.